### PR TITLE
Added support for Django 2.0 and py3.6

### PR DESCRIPTION
**Pull request by jarshwah**

### CODE REVIEW

File: box-tests.sh [hunk 1]
```diff
@@ -28,11 +21,17 @@ runtests35-sqlite3 auth_tests
...
 
-runtests27-sqlite3 admin_widgets --selenium chrome
+runtests36-sqlite3-gis gis_tests
+runtests36-postgres-gis gis_tests
+runtests36-mysql-gis gis_tests
+runtests36-sqlite3 auth_tests
+runtests36-postgres auth_tests
+runtests36-mysql auth_tests
+
+runtests36-sqlite3 admin_widgets --selenium chrome --parallel 1
```
Comment (timgraham) [inline box-tests.sh:31]: There's no need to include admin_widgets here (that doesn't include  all selenium tests). `--selenium` limits to the selenium tests.

File: README.md [hunk 2]
```diff
@@ -145,7 +153,9 @@ Chrome is pre-installed so that Django's selenium tests can be run in headless
 mode with a virtual display (id `:99`). For example, you may run a specific test
 like so:
 
-    (vm) $ runtests27-sqlite3 admin_widgets --selenium chrome
+    (vm) $ runtests36-sqlite3 admin_widgets --selenium chrome --parallel 1
+
+The test suite will sometimes hang when running selenium tests in parallel mode.
```
Comment (timgraham) [inline README.md:158]: I haven't seen this issue before but did reproduce it on the master branch just now. Did you see it on the 1.11 branch? I'm wondering if it's caused by a recent change in Django.

File: box-tests.sh [hunk 3]
```diff
@@ -28,11 +21,17 @@ runtests35-sqlite3 auth_tests
 runtests35-postgres auth_tests
 runtests35-mysql auth_tests
 
-runtests27-sqlite3 admin_widgets --selenium chrome
+runtests36-sqlite3-gis gis_tests
```
Comment (timgraham) [inline box-tests.sh:24]: It may help to use the same terminology as Jenkins: spatialite, postgis, mysql_gis

File: README.md (hunk 2 above)
Comment (jarshwah) [reply README.md:158]: No, I don't remember seeing this on the 1.11 branch. I'd planned to go back and test to see if we needed to open a ticket against Django.

Did you reproduce on master with this box, or another setup?

File: box-tests.sh (hunk 1 above)
Comment (jarshwah) [reply box-tests.sh:31]: I don't want to execute every selenium test - just a few of them to validate that selenium testing works in general.

Comment (timgraham) [reply box-tests.sh:31]: Oops, didn't notice this was `box-tests.sh`.

File: box-tests.sh (hunk 3 above)
Comment (jarshwah) [reply box-tests.sh:24]: Good idea - being consistent in the naming will help if I'm promoting the similarities between the environments. It'll also force me to cleanup the current ansible statements that generate the aliases which aren't pretty at all.

File: README.md (hunk 2 above)
Comment (timgraham) [reply README.md:158]: Reproduced without the box. I'm suspecting https://github.com/django/django/commit/bece837829eafbc22f2598dadf82c9a8364b085a might be the culprit but am running repeated tests on the commit before to see if reproduces there.

Comment (jarshwah) [reply README.md:158]: If it helps, I found the tests hanging on the last test. I'm guessing it has something to do with the shutdown procedure.

**Pull request merged**